packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SVGComponents/VXZoom/VXZoom.js

```diff
@@ -62,6 +69,67 @@ class VXZoom extends PureComponent {
     }
   }
 
+  onPan (direction) {
+    const { 
+      transformMatrix: {
+        translateX,
+        translateY
+      }
+    } = this.zoom
+    const panDistance = 20
+    const newTransformation = { 
+      down: { translateX, translateY: translateY - panDistance },
+      left: { translateX: translateX + panDistance, translateY },
+      right: { translateX: translateX - panDistance, translateY },
+      up: { translateX, translateY: translateY + panDistance }
+    }
```

Comment:
Since `dx` and `dy` are passed to `onPan`, the new transformation can be calculated from `translateX + dx * panDistance, translateY + dy * panDistance` for all four directions.